react-native-reanimated.d.ts

```diff
@@ -242,7 +242,7 @@ declare module 'react-native-reanimated' {
       getNode(): ReactNativeScrollView;
     }
     export class Code extends Component<CodeProps> {}
-    export function createAnimatedComponent<S extends object, P extends { style?: StyleProp<S>; }>(component: ComponentType<P>): ComponentType<AnimateProps<S, P>>;
+    export function createAnimatedComponent<S extends object, P extends { style?: StyleProp<S>; }>(component: ComponentClass<P>): ComponentClass<AnimateProps<S, P>>;
```

Comment:
```suggestion
    export function createAnimatedComponent<S extends object, P extends { style?: StyleProp<S>; } | {}>(component: ComponentClass<P>): ComponentClass<AnimateProps<S, P>>;
```

That way we can use react-native-svgs `PathProps` without style attribute.